crates/router/src/connector/noon/transformers.rs

```diff
@@ -161,7 +161,7 @@ pub struct NoonPayPal {
 }
 
 #[derive(Debug, Serialize)]
-#[serde(tag = "type", content = "data")]
+#[serde(tag = "type", content = "data", rename_all = "UPPERCASE")]
```

Comment:
Please make sure this won't affect any of the payment methods

Reply:
It doesn't effect any other pmd